src/net/udp.rs

```diff
@@ -1,9 +1,9 @@
 use std::net::{self, ToSocketAddrs};
 
-
-use packet::{Packet, PacketProcessor};
-use net::{NetworkConfig, SocketState};
 use error::{NetworkError, Result};
+use events::Event;
+use net::{NetworkConfig, SocketState};
+use packet::{Packet, PacketProcessor};
 
 /// Maximum amount of data that we can read from a datagram
 const BUFFER_SIZE: usize = 1024;
```

Comment:
We don't need to do it now, but we should probably make an issue to make this configurable.